2021/23xxx/CVE-2021-23334.json

```diff
@@ -1,97 +1,99 @@
 {
```

Comment:
{} is not allowed for {'vendor': {'vendor_data': [{'vendor_name': 'n/a', 'product': {'product_data': [{'product_name': 'static-eval', 'version': {'version_data': [{'version_name': '', 'version_affected': '>=', 'version_value': '0', 'platform': ''}]}}]}}]}}